### ISSUE TITLE
Unintuitive messages in the log during index creation or delete commands

### ISSUE THREAD

**Issue by santo-it** [orientechnologies/orientdb#6889]
## OrientDB Version, operating system, or hardware.
- v. 2.2.12

## Operating System
- [ ] Windows

---

Hi,

When creating an index, e.g. with the following code:

```
Collection<OClass> classCollection = oDb.getRawGraph().getMetadata().getSchema().getClass("V").getAllSubclasses();
for (OClass classCollectionElement : classCollection) {
    
	oDb.getRawGraph().getMetadata().getSchema().getClass(ClassCollectionElement.getName()).createProperty("aProperty",OType.LONG);                    
    oDb.getRawGraph().getMetadata().getSchema().getClass(ClassCollectionElement.getName()).getProperty("aProperty").createIndex(OClass.INDEX_TYPE.UNIQUE);          
}
```

some unintuitive messages are written in the log, e.g. 

```
ott 25, 2016 7:16:19 PM com.orientechnologies.common.log.OLogManager log
INFORMAZIONI: $ANSI{green {db=aDatabase}} [TIP] Query 'SELECT count(*) FROM `Address` WHERE `aProperty`.type() NOT IN ["LONG", "SHORT", "INTEGER", "BYTE", "BOOLEAN"] AND `aProperty` IS NOT NULL' fetched more than 50000 records: to speed up the execution, create an index or change the query to use an existent index
ott 25, 2016 7:16:26 PM com.orientechnologies.common.log.OLogManager log
INFORMAZIONI: $ANSI{green {db=aDatabase}} [TIP] Query 'SELECT FROM `Address` WHERE `aProperty`.type() <> "LONG"' fetched more than 50000 records: to speed up the execution, create an index or change the query to use an existent index
ott 25, 2016 7:16:29 PM com.orientechnologies.common.log.OLogManager log
INFORMAZIONI: $ANSI{green {db=aDatabase}} - Rebuilding index aDatabase.Address.aProperty (estimated 151.592 items)...
ott 25, 2016 7:16:39 PM com.orientechnologies.common.log.OLogManager log
INFORMAZIONI: $ANSI{green {db=aDatabase}} --> 75,06% progress, 113.780 indexed so far (11.378 items/sec)
ott 25, 2016 7:16:43 PM com.orientechnologies.common.log.OLogManager log
INFORMAZIONI: $ANSI{green {db=aDatabase}} --> OK, indexed 151.592 items in 13.530 ms
ott 25, 2016 7:16:45 PM com.orientechnologies.common.log.OLogManager log
```

These messages can confuse new users. They make you think that you have executed a SELECT FROM - but this is not the case. The messages suggest to "create an index or change the query to use an existent index" - but creating an index is right what the user is doing 

Please let's improve these messages

Thanks,

**Comment by santo-it** [comment]
This can happen also when executing a DELETE - as an internal select is used to load the records that have to be deleted and that SELECT FROM .. message in the log can confuse users as a DELETE was executed and the SELECT was internal.

**Comment by luigidellaquila** [comment]
Fixed in 2.2.x branch, the fix will be released with 2.2.23

Thanks

Luigi
